thoth/storages/graph/postgres.py

```diff
@@ -4104,7 +4090,7 @@ def _python_sync_analysis_result(
                 os_version=software_environment.os_version,
                 python_version=software_environment.python_version,
                 index_url=None,
-                sync_only_entity=True
+                is_external=True
```

Comment:
This depends on the actual image analysis run - if it is external or analysis of an image that we plan to use in recommendations- there should be a metadata field propagated - https://github.com/thoth-station/user-api/blob/967285e46893f68d79b7f4ae5da81ed6c50b0e9d/thoth/user_api/api_v1.py#L82